RecoLocalCalo/HGCalRecAlgos/interface/HGCalRecHitSimpleAlgo.h

```diff
@@ -41,7 +41,8 @@ class HGCalRecHitSimpleAlgo : public HGCalRecHitAbsAlgo {
     //    float clockToNsConstant = 25;
     float energy = uncalibRH.amplitude() * adcToGeVConstant_;
     float time   = uncalibRH.jitter();
-    if(time<0) time   = 0; // fast-track digi conversion
+
+    //if(time<0) time   = 0; // fast-track digi conversion
```

Comment:
So, will HGCRecHit constructor "throw", or do you have to make sure right inputs go in?